Packs/PrismaSaasSecurity/Integrations/SaasSecurity/README.md

```diff
@@ -18,7 +18,7 @@ Use the SaaS Security integration to protect against cloud‑based threats by:
     | Incidents Fetch Interval | Frequency \(in hours and minutes\) by which Cortex XSOAR fetches incidents from SaaS Security when **Fetch Incidents** is selected. | False |
     | Incident type | Incident type is set by this field if a classifier does not exist. If a  classifier is selected, it takes precedence. | False |
     | Incident Mirroring Direction | Selects which direction you want the incidents mirrored. You can mirror Incoming only \(from SaaS Security to Cortex XSOAR\), **Outgoing** only \(from Cortex XSOAR to SaaS Security\), or both **Incoming And Outgoing**. | False |
-    | Number of incidents per fetch | Minimum is 10. Maximum is 1000. | True |
+    | Number of incidents per fetch | Important: The limit value can range from 10 to 200 and must be in multiples of 10. | True |
```

Comment:
Do you need to say that the default is 50?

Reply:
No, it appears clearly to the user